### PR TITLE
expose the index name

### PR DESCRIPTION
**Pull request by flepied**
- allow to specify the name of the index in the projects.yaml file
- all the URL are prefixed by the index name

Issue: https://github.com/morucci/monocle/issues/17

### CODE REVIEW

File: web/src/components/filtersform.js [hunk 1]
```diff
@@ -109,12 +118,22 @@ class FiltersForm extends React.Component {
...
+                    value={this.props.filter_index}
+                    onChange={this.onRepositoryChange}
+                  />
+                </Form.Group>
+              </Col>
                 <Form.Group controlId="formRepositoryInput">
                   <Form.Label>Repository regexp</Form.Label>
                   <Form.Control
                     type="text"
                     value={this.props.filter_repository}
```
Comment (morucci) [inline web/src/components/filtersform.js:135]: I think that inverted here

File: web/src/components/filtersform.js [hunk 2]
```diff
@@ -109,12 +118,22 @@ class FiltersForm extends React.Component {
           <Form onSubmit={this.handleSubmit}>
             <Form.Row>
               <Col>
+              <Col>
+                <Form.Group controlId="formIndexInput">
+                  <Form.Label>Index name</Form.Label>
+                  <Form.Control
+                    type="text"
+                    value={this.props.filter_index}
```
Comment (morucci) [inline web/src/components/filtersform.js:126]: I think that inverted here

Comment (morucci) [reply web/src/components/filtersform.js:126]: I the UI I can type in "Repository regexp". I guess due to that.

File: web/src/components/filtersform.js (hunk 1 above)
Comment (flepied) [reply web/src/components/filtersform.js]: fixed

File: web/src/components/filtersform.js (hunk 2 above)
Comment (flepied) [reply web/src/components/filtersform.js]: fixed

**Comment by flepied** [comment]
@morucci could we merge it now?

**Pull request merged**